exports.go

```diff
@@ -34,14 +34,24 @@ const (
 	ResponseTypeChunked = "Chunked"
 )
 
+// ServiceLatency is used when observing and exported service for
+// latency measurements.
+// Sampling 1-100, represents sampling rate, defaults to 100.
+// Results is the subject where the latency metrics are published.
```

Comment:
Do we want to say what the format of the metrics are and what interval they are published on?

Reply:
They are linked to the server, but agree we should mention. They are published as they happen.

Reply:
Will fix.